PasteBinService/RegisterService/src/main/java/org/potrohit/springboot/RegisterService/activity/UpdatePasteBinActivity.java

```diff
@@ -0,0 +1,23 @@
+package org.potrohit.springboot.RegisterService.activity;
+
+import org.potrohit.springboot.RegisterService.activity.common.RegisterPasteBinRequest;
+import org.potrohit.springboot.RegisterService.activity.common.PasteBinResponse;
+import org.potrohit.springboot.RegisterService.activity.common.RequestMappings;
+import org.potrohit.springboot.RegisterService.domain.bussinesslogic.UpdatePasteBinService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping
+public class UpdatePasteBinActivity {
+    @Autowired
+    private UpdatePasteBinService updatePasteBinService;
```

Comment:
these should be final object. We should not allow any one to reassign it to a new object